pkg/sqlfs/writer.go

```diff
@@ -23,7 +23,8 @@ import (
 	pb "sqlflow.org/sqlflow/pkg/proto"
 )
 
-const bufSize = 4 * 1024
+// TEXT/STRING field support 64KB maximumn storage size
+const bufSize = 32 * 1024
```

Comment:
Can we make this `bufSize` configurable? If my memory serves me right, the `string` type in MaxCompute has a much bigger size than 64KB.

Reply:
Better to keep it constant for better saved model migrations.